src/pages/docs/liveobjects/concepts/objects.mdx

```diff
@@ -94,11 +97,12 @@ userSettings.set("notifications", LiveMapValue.of(true));
 
 <Code>
 ```javascript
-// Create a LiveCounter
-const visitsCounter = await channel.objects.createCounter();
+// Create and assign a LiveCounter
+await myObject.set('visits', LiveCounter.create(0));
 
-// Increment the counter
-await visitsCounter.increment(1);
+// Access via path
+const visits = myObject.get('visits');
```

Comment:
I have a similar question here to above; in this scenario, is it possible that the counter that gets incremented here is not the one that they created in the `myObject.set('visits', LiveCounter.create(0))` call?